docs/powergate/cli/pow_admin_jobs_latest-successful.md

```diff
@@ -0,0 +1,32 @@
+## pow admin jobs latest-successful
```

Comment:
Idea: Maybe this command can be an extra flag on `pow admin job latest-final`.

Reply:
will create an issue in powergate

Reply:
https://github.com/textileio/powergate/issues/708